api/v1beta2/tenantresource_namespaced.go

```diff
@@ -17,7 +17,7 @@ type TenantResourceSpec struct {
 	// When the replicated resource manifest is deleted, all the objects replicated so far will be automatically deleted.
 	// Disable this to keep replicated resources although the deletion of the replication manifest.
 	// +kubebuilder:default=true
-	PruningOnDelete *bool `json:"pruningOnDelete,omitempty"`
+	PruningOnDelete bool `json:"pruningOnDelete,omitempty"`
```

Comment:
The pointer here must be kept, otherwise the default value `true` will never be applied since the default value of a boolean is `false`.

Reply:
done